web/docs/reference.mdx

```diff
@@ -0,0 +1,466 @@
+---
+id: reference
+title: Reference Documentation
+sidebar_label: Overview
+description: 'Reference Docs'
+hide_table_of_contents: true
+---
+
+import Link from '@docusaurus/Link'
+
+The Supabase Reference Docs provide technical descriptions of the products and how to use them.
+
+<div class="container" style={{ padding: 0 }}>
+  <div class="row is-multiline">
+    {/* Examples - coming soon */}
+    {/* <div class="col col--6">
+      <Link class="card" to="/docs/guides/database" style={{ height: '100%' }}>
+        <div class="card__body">
+          <h4>Examples</h4>
+          <p>Code examples and snippets for Supabase products.</p>
+        </div>
+      </Link>
+    </div> */}
+    {/* CLI */}
+    <div class="col col--6">
+      <Link class="card" to="/docs/reference/cli/about" style={{ height: '100%' }}>
+        <div class="card__body">
+          <h4>CLI</h4>
+          <p>Manage your Supabase projects from the command line.</p>
+        </div>
+      </Link>
+    </div>
+    {/*  */}
+  </div>
+</div>
+
+## Client Libraries
+
+<div class="container" style={{ padding: 0 }}>
+  <div class="row is-multiline">
+    {/* Database */}
+    <div class="col col--6">
+      <Link class="card" to="/docs/reference/javascript/installing" style={{ height: '100%' }}>
+        <div class="card__body">
+          <h4>Javascript Client Library</h4>
```

Comment:
I think its a little redundant to restate that Javascript is a client library, and if we want to restate that its a client library then we should do the same for Dart.

Reply:
good call!